<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="0">
    <layout class="QFormLayout" name="formLayout_6">
     <item row="0" column="0">
      <widget class="QCheckBox" name="minimizeBox">
       <property name="toolTip">
        <string>Enable this to automatically minimize after connection</string>
       </property>
       <property name="text">
        <string>Minimize on Connect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="batchModeBox">
       <property name="toolTip">
        <string>Enable this to remember passwords for this server</string>
       </property>
       <property name="text">
        <string>Batch mode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="disableUDP">
       <property name="text">
        <string>Disable UDP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="proxyBox">
       <property name="toolTip">
        <string>Enable this to use the system's default proxy settings for this server</string>
       </property>
       <property name="text">
        <string>Use Proxy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="labelEdit">
         <property name="toolTip">
          <string>This is a friendly name to distinguish the connection</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Gateway</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="gatewayEdit">
         <property name="toolTip">
          <string>Specify the hostname to connect to; a port may be specified after the host separated with a colon ':'</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="usernameEdit">
         <property name="toolTip">
          <string>Specify the username used in that gateway</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Groupname</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="groupnameEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/new/resource/images/trashcan.png</normaloff>:/new/resource/images/trashcan.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>CA Certificate</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="caCertButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="caCertEdit">
           <property name="toolTip">
            <string>Specify the CA certificate, if any; by default the system CAs will be used</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If not set, the system's trusted certificate storage will be used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="caCertClear">
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/new/resource/images/trashcan.png</normaloff>:/new/resource/images/trashcan.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Server Certificate</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="serverCertHash">
           <property name="toolTip">
            <string>This is the hash of the server's certificate</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="serverCertClear">
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/new/resource/images/trashcan.png</normaloff>:/new/resource/images/trashcan.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="caCertHash">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>OTP Token</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="tokenBox">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="maxVisibleItems">
            <number>3</number>
           </property>
           <item>
            <property name="text">
             <string notr="true">HOTP (RFC4226)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">TOTP (RFC6238)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>STOKEN (RSA)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tokenEdit">
           <property name="toolTip">
            <string>This is the token code; should be a hex value (starting with 0x), or a value starting with 'base32:'</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tokenClear">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/new/resource/images/trashcan.png</normaloff>:/new/resource/images/trashcan.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="11" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Local Certificate</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>User Certificate</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QToolButton" name="userCertButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="userCertEdit">
               <property name="toolTip">
                <string>Specify your certificate file, if any</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="userCertClear">
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/new/resource/images/trashcan.png</normaloff>:/new/resource/images/trashcan.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="userCertHash">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="userKeyLabel">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>User Key</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QToolButton" name="userKeyButton">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="userKeyEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="toolTip">
                <string>Specify your key file if any; in case of a PKCS #12 file, you only need to set the certificate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="userKeyClear">
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/new/resource/images/trashcan.png</normaloff>:/new/resource/images/trashcan.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>System Store</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QListWidget" name="loadWinCertList"/>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="loadWinCert">
           <property name="toolTip">
            <string extracomment="Select that certificate"/>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
